packages/frontend/src/components/common/LoadingChart.tsx

```diff
@@ -0,0 +1,28 @@
+import { LOADING_CHART_CLASS } from '@lightdash/common';
+import { clsx } from '@mantine/core';
+import { type FC, type ReactNode } from 'react';
+import SuboptimalState from './SuboptimalState/SuboptimalState';
+type LoadingChartProps = {
+    children?: ReactNode;
+    className?: string;
+};
```

Comment:
NP: could you add a new line between 4-5 ?  
  
Also... do you think it needs `children` prop?